source/speech/__init__.py

```diff
@@ -790,7 +826,25 @@ def _speakTextInfo_addMath(speechSequence, info, field):
 	except (NotImplementedError, LookupError):
 		return
 
-def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=controlTypes.REASON_QUERY, _prefixSpeechCommand=None, onlyInitialFields=False, suppressBlanks=False,priority=None):
+
+def _logBadSequenceTypes(sequence, shouldRaise: bool = True) -> bool:
+	return types.logBadSequenceTypes(sequence, raiseExceptionOnError=shouldRaise)

```

Comment:
Ah, so it turns out that at the top of the file, my assumption that you were importing Python core types module was incorrect. That has to be from . import types, then.